### ISSUE TITLE
V2 resets properties to 0 before animating

### ISSUE THREAD

**Issue by yura3d** [julianshapiro/velocity#852]
* VelocityJS version: 2.0.1
* Browser: Chrome 64.0.3282.186 x64, Firefox 58.0.2 x64
* Operating System: Win10 Pro x64 1709 16299.248

Hello!
After updating from 1.x.x to 2.0.1 I've noticed, that velocityjs resets animation properties (for example, height property) to 0 (zero) before starting an animation.
My JSFiddle: https://jsfiddle.net/xhkey2ty/1/
In this JSFiddle, when I'm clicking on a bottom link, I'm expecting that the height of orange block will changing from its current value (100px) to target value (500px). But velocityjs starts the animation by reseting the height to 0 and after that animates the height from 0 to 500px. But this is not what I'm expecting. You can change "velocity" method in my JSFiddle to jQuery "animate" method, and it is working as expected, changing the height from 100px to 500px correctly.

**Comment by Rycochet** [comment]
I'm tweaking how things are happening, but this is a case of user error - you're animating from `100px` to `500` - note the lack of a "px" on the end of the second.

Velocity has gone from animating a small section of the available css properties, to being able to animate all css properties and svg (and other) attributes - so you should be passing valid css to it when trying to animate some css.

This is one of the shortcuts that we're wanting to try and leave in, but it's always better to supply the correct values to begin with.

Please note that the new issue template is not optional for support.
